src/util/defaults.ml

```diff
@@ -178,8 +178,10 @@ let _ = ()
       ; reg Experimental "exp.fast_global_inits" "false" "Only generate 'a[0] = 0' for a zero-initialized array a[n]. This is only sound for our flat array domain! TODO change this once we use others!"
       ; reg Experimental "exp.uninit-ptr-safe"   "false" "Assume that uninitialized stack-allocated pointers may only point to variables not in the program or null."
       ; reg Experimental "exp.ptr-arith-safe"    "false" "Assume that pointer arithmetic only yields safe addresses."
+      ; reg Experimental "exp.witness_path" "'witness.graphml'" "Witness output path"
       ; reg Experimental "exp.minwitness"        "false" "Try to minimize the witness"
       ; reg Experimental "exp.uncilwitness"      "false" "Try to undo CIL control flow transformations in witness"
+      ; reg Experimental "exp.architecture"      "'64bit'" "Architecture for analysis, currently for witness"
```

Comment:
This should be tied in with the CIL flags concerning architecture, rather than being specifiable on its own. See also https://github.com/goblint/analyzer/issues/54